golang/testsuite/networks_impl/test_network.go

```diff
@@ -121,13 +133,59 @@ func (network *TestNetwork) addApiService() (*api_service_client.APIClient, erro
 	network.nextApiServiceId = network.nextApiServiceId + 1
 	serviceId := services.ServiceID(serviceIdStr)
 
-	configFactory := api.NewApiContainerConfigFactory(network.apiServiceImage, network.datastoreClient)
-	apiServiceContext, hostPortBindings, err := network.networkCtx.AddService(serviceId, configFactory)
+	type config struct {
```

Comment:
Same with this - think it's worth extracting all this API initialization stuff into a helper function, so it's more obvious what's happening